design-documents/asynchronous-import/base-extension.md

```diff
@@ -24,13 +24,10 @@ This request can accept files from different sources:
 
 ```
 {
-  "fileEntry": {
-    "file_id": 0,
-    "source": {
+  "source": {
       "import_data": "/var/www/html/bulk-api/async-import/var/catalog_product.csv",
```

Comment:
We shouldn't support importing from the outside of  Magento root

Reply:
Thanks, its true